a10_neutron_lbaas/tests/unit/v1/test_handler_member.py

```diff
@@ -113,3 +116,12 @@ def test_delete_count_gt_one(self):
         pool_name = self.a.member._pool_name(None, m['pool_id'])
         self.a.last_client.slb.service_group.member.delete.assert_called_with(
             pool_name, name, m['protocol_port'])
+
+    def test_updating_oper_stats(self):
+        status_check.status_update_v1(self.a)
+        self.a.last_client.slb.service_group.member.get_oper.mock_calls[0].assert_called_with(
```

Comment:
If `get_oper` has no `mock_calls` made to it, this line will fall on it's face.  This needs a preliminary length check and/or ternary operator assignment.